lib/smooth/application/io/i2c/DS3231.cpp

```diff
@@ -0,0 +1,658 @@
+// Smooth - C++ framework for writing applications based on Espressif's ESP-IDF.
```

Comment:
As the license is changing, please update the header to use the Apache 2.0 notice as outlined in the new CONTRIBUTING.md